docs/vereisten/automatische_logregistratie.md

```diff
@@ -17,7 +17,8 @@ bouwblok:
 <!-- tags -->
 ## Vereiste
 
-AI-systemen met een hoog risico zijn dusdanig technisch vormgegeven dat gebeurtenissen gedurende hun levenscyclus automatisch worden geregistreerd (“logs”).
+AI-systemen met een hoog risico zijn dusdanig technisch vormgegeven dat gebeurtenissen gedurende hun levenscyclus automatisch worden geregistreerd (“logs”). Voor alle AI-systemen bij bestuursorganen is hoofdstuk 12.4 van de BIO van toepassing met generieke verplichtingen tot het loggen en monitoren. Voor zover het AI-systeem persoonsgegevens verwerkt zorgen artikel 5 en 32 van de AVG ook voor een verplichting om verwerkingen te loggen en monitoren.
```

Comment:
```suggestion
Algoritmes en AI-systemen zijn dusdanig technisch vormgegeven dat gebeurtenissen gedurende hun levenscyclus automatisch worden geregistreerd (“logs”). 
```

Reply:
In de tekst van de vereiste liever geen verwijzingen naar wetgevingen. Deze staan bij de bronnen. Tekst dusdanig aangepast dat deze niet alleen geldt voor hoog risico AI systemen.